client/modules/core/main.js

```diff
@@ -387,10 +387,9 @@ export default {
   },
 
   getUserPreferences(packageName, preference, defaultValue) {
-    const user = Meteor.user();
-
+    const user = Accounts.findOne(Meteor.userId());
```

Comment:
consider adding a comment so that this change is not reversed in the future

Reply:
@pmn4 Thanks, I have added a comment where the user preferences are being saved.